org.openmw.OpenMW.json

```diff
@@ -0,0 +1,360 @@
+          "-DMYGUI_BUILD_PLUGINS=0"
+        ],
+        "sources": [
+          {
+            "type": "archive",
+            "url": "https://github.com/MyGUI/mygui/archive/MyGUI3.2.2.tar.gz",
+            "sha256": "0a28d7ec8a47993cb68deb48b36331e28f12dd92580b709eaef21d599b67a78f"
+          }
+        ],
+        "cleanup": [
+          "/include",
+          "/lib/*.a",
+          "/lib/*.la",
+          "/lib/pkgconfig"
+        ]
+      },
+      {
+        "name": "libunshield",
+        "buildsystem": "cmake",
+        "sources": [
+          {
+            "type": "archive",
+            "url": "https://github.com/twogood/unshield/archive/1.4.2.tar.gz",
+            "sha256": "5dd4ea0c7e97ad8e3677ff3a254b116df08a5d041c2df8859aad5c4f88d1f774"
+          }
+        ],
+        "cleanup": [
+          "/include",
+          "/lib/*.a",
+          "/lib/*.la",
+          "/lib/pkgconfig"
+        ]
+      },
+      {
+        "name": "boost",
+        "sources": [
+          {
+            "type": "archive",
+            "url": "https://downloads.sourceforge.net/project/boost/boost/1.64.0/boost_1_64_0.tar.bz2",
+            "sha256": "7bcc5caace97baa948931d712ea5f37038dbb1c5d89b43ad4def4ed7cb683332"
+          },
+          {
+            "type": "script",
+            "commands": [
+              "./bootstrap.sh \"$@\"",
+              "cat <<EOF >Makefile",
+              "all:",
+              "\t./b2 -j `nproc` ./b2 stage threading=multi link=shared",
+              "",
+              "install:",
+              "\t./b2 install",
+              "EOF"
+            ],
+            "dest-filename": "configure"
+          }
+        ],
+        "cleanup": [
+          "/include",
+          "/lib/*.a",
+          "/lib/*.la",
+          "/lib/pkgconfig",
+          "libboost_atomic*",
+          "libboost_chrono*",
+          "libboost_container*",
+          "libboost_context*",
+          "libboost_coroutine*",
+          "libboost_date_time*",
+          "libboost_fiber*",
+          "libboost_graph*",
+          "libboost_iostreams*",
+          "libboost_locale*",
+          "libboost_log*",
+          "libboost_math*",
+          "libboost_prg_exec_monitor*",
+          "libboost_python*",
+          "libboost_random*",
+          "libboost_regex*",
+          "libboost_serialization*",
+          "libboost_signals*",
+          "libboost_thread*",
+          "libboost_timer*",
+          "libboost_type_erasure*",
+          "libboost_unit_test_framework*",
+          "libboost_wave*",
+          "libboost_wserialization*"
+        ]
+      },
+      {
+        "name": "openmw",
+        "builddir": true,
+        "buildsystem": "cmake",
+        "config-opts": [
+          "-DBUILD_BSATOOL=no",
+          "-DBUILD_ESMTOOL=no",
+          "-DBUILD_MYGUI_PLUGIN=no"
+        ],
+        "sources" : [
+          {
+            "type": "git",
+            "url": "git://github.com/OpenMW/openmw.git",
+            "branch": "openmw-0.42.0"
```

Comment:
Can this get a commit or a tag as well to specify the release more tightly. 

It'll be faster to build if you just switch it over to the source code archive [published here](https://github.com/OpenMW/openmw/releases).